improver/calibration/reliability_calibration.py

```diff
@@ -437,3 +437,74 @@ def process(self, historic_forecasts, truths):
             reliability_tables.append(reliability_entry)
 
         return MergeCubes()(reliability_tables)
+
+
+class AggregateReliabilityCalibrationTables:
+
+    """This plugin enables the aggregation of multiple reliability calibration
+    tables, and/or the aggregation over coordinates in the tables."""
+
+    def __init__(self):
+        """
+        Initialise plugin for manipulating reliability calibration tables.
+        This plugin can aggregate multiple tables or aggregate over coordinates
+        using summation.
+        """
+        self.cube_index_coord = 'cube_coord'
+        self.merge_coord = lambda value: iris.coords.DimCoord(
+            np.array([value]).astype(np.int32),
+            long_name=self.cube_index_coord, units=1)
+
+    def __repr__(self):
+        """Represent the configured plugin instance as a string."""
+        return '<AggregateReliabilityCalibrationTables>'
+
+    def _construct_single_cube(self, cubes):
```

Comment:
It might be worth reviewing the metadata available for combining reliability calibration tables, given that this method is assuming that the metadata will make the tables indistinguishable yet that they still require combining. I would see the indistinguishability of the tables, in terms of their metadata, as a bit of a concern, so perhaps we should revisit this. What do you think?